map/bookmark_helpers.cpp

```diff
@@ -245,15 +245,15 @@ std::string RemoveInvalidSymbols(std::string const & name)
   return strings::ToUtf8(filtered);
 }
 
-std::string GenerateUniqueFileName(const std::string & path, std::string name, std::string const & ext)
+std::string GenerateUniqueFileName(std::string const & path, std::string name, std::string_view const ext)
```

Comment:
No need here. Instead of using _one-time_ created string constants, now we create std::string(ext) object every time when calling this fucntion.

Reply:
In this case what is a valid way to change type of extensions if we have

std::string GenerateUniqueFileName(const std::string & path, std::string name, std::string_view ext = kKmlExtension);

where kKmlExtension is a string_view? How it should be converted to a string to be concatenated inside GenerateUniqueFileName?

Reply:
Doesn't string.append() work?